sdk/src/Services/S3/Custom/Transfer/TransferUtilityUploadDirectoryRequest.cs

```diff
@@ -259,6 +263,58 @@ public string ServerSideEncryptionKeyManagementServiceKeyId
             set { this.serverSideEncryptionKeyManagementServiceKeyId = value; }
         }
 
+        /// <summary>
+        /// The Server-side encryption algorithm to be used with the customer provided key.
+        ///  
```

Comment:
Minor, but could you remove this blank line? Unsure how it'll flow through to the docs.

Reply:
Thank you for the review. Removed the empty line.